Configuration/Settings.yaml

```diff
@@ -13,22 +13,35 @@ Neos:
 
   Neos:
     Setup:
-      #
-      # Imagine drivers that are supported
-      #
+      # ordering: from "least-fitting" to "best-fitting". We will suggest the best matching one LAST in the list.
```

Comment:
I am wondering whether we should make the supported image handlers and file types configurable, or move that into Code. My preference would be code, because I do not see any use case for users to override it (besides it being impossible to override cleanly because it is a numbered array only).

Reply:
or let our infamous positional array sorter do the rest:)
e.g. keep it as is, plus add the `position` keys and thus its already more explicit and extensible.

(tbh i think the extensibility here is yagni but it might be nice for us as maintainers to better read the code?)

Reply:
I'd rather like to remove the extensibility and make the options a static class property :) if you do not object

Reply:
thats a good idea:) it was just historically from the old GUI setup. Having these options internal is much more sane as it will be hard or impossible to actually extend them:)

Reply:
I will publish an update tonight. Currently on holidays 😬